src/views/Project/index.jsx

```diff
@@ -111,12 +130,22 @@ export default class Project extends Component {
       [];
 
     return (
-      <Fragment>
-        <header>
-          <Typography variant="display2" align="center">
+      <div className={classes.root}>
+        <AppBar position="absolute" className={classes.header}>
+          <Typography
+            variant="display2"
+            align="left"
+            className={classes.link}
+            component={NavLink}
+            to="/">
+            Codetribute
+          </Typography>
+        </AppBar>
```

Comment:
What do you think of integrating the project name in the AppBar:

![screen shot 2018-06-22 at 2 39 32 pm](https://user-images.githubusercontent.com/3766511/41793507-561a8eb2-762a-11e8-828b-9f03fcc13940.png)

Reply:
:+1 wah it looks much better